controllers/apply-next/form-router.js

```diff
@@ -116,32 +139,60 @@ function initFormRouter(formModel) {
     });
 
     /**
-     * Require login, redirect back here once authenticated.
-     */
-    router.use(requireUserAuth);
-
-    /**
-     * Set common locals
+     * Route: Eligibility checker
      */
-    router.use(async (req, res, next) => {
-        // Translate the form object for each request and populate it with current user input
-        const form = enhanceForm({
-            locale: req.i18n.getLocale(),
-            baseForm: formModel
-        });
+    router
```

Comment:
The eligibility step now no longer requires people to be logged in. It also no longer stores progress in the session, so acts as a soft barrier to entry rather than being hard enforced.